test/CMakeLists.txt

```
@@ -1231,6 +1233,20 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_viirs_aod
                           -n 0.0"
                           viirs_aod.nc ${IODA_CONV_COMP_TOL})
 
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_viirs_aod
```

```suggestion
ecbuild_add_test( TARGET  test_${PROJECT_NAME}_viirs_smoke_dust
```

this test should have a unique name